### DISCUSSION TITLE
sonicjs looks awesome :) can we use bootstrap 5?

### DISCUSSION THREAD

**Discussion by lindamacdonald** [SonicJs-Org/sonicjs#42] [Ideas]
Hey there folks this looks amazing :) ;) any chance we can use bootstrap 5? Bootstrap is getting rid of jquery as a dependency and I presume it will therefore be super for loadspeed? :)

**Comment by lane711** [comment]
Yes, that is in our backlog:
https://github.com/lane711/sonicjs/issues/48

Thanks for your interest!

**Comment by lane711** [comment]
Bootstrap 5 is now implemented in SonicJs!
